### PR TITLE
Basic diagnostics added

### PR DESCRIPTION
**Pull request by surirohit**

### CODE REVIEW

File: src/pi_sonar.cpp [hunk 1]
```diff
@@ -164,22 +183,44 @@ int main(int argc, char *argv[])
 
   ROS_INFO("Pi Sonar node ready");
 
+  diagnostic_updater::Updater updaters[5];
```
Comment (rohbotics) [inline src/pi_sonar.cpp]: doesn't make sense to add this to every sonar object, instead of an external array?

File: src/pi_sonar.cpp [hunk 2]
```diff
@@ -164,22 +183,44 @@ int main(int argc, char *argv[])
...
+  diagnostic_updater::HeaderlessTopicDiagnostic *pub_freq[5];
```
Comment (rohbotics) [inline src/pi_sonar.cpp]: Again move to sonar class, and use unique_ptr instead of raw ptr.

**Review by rohbotics — CHANGES_REQUESTED** [review]

File: src/pi_sonar.cpp (hunk 1 above)
Comment (surirohit) [reply src/pi_sonar.cpp]: Yes, it does. I mentioned in my message that I would do it in the next update.

File: src/pi_sonar.cpp (hunk 2 above)
Comment (surirohit) [reply src/pi_sonar.cpp]: Yes. I'll do that.

**Comment by surirohit** [comment]
Updated as requested!

**Review by rohbotics — APPROVED** [review]

**Pull request merged**